src/main/java/edu/ucdavis/dss/ipa/api/components/budget/BudgetViewController.java

```diff
@@ -144,6 +146,30 @@ public LineItem createLineItem(@PathVariable long budgetScenarioId,
         return lineItem;
     }
 
+    @RequestMapping(value = "/api/budgetView/budgets/{budgetId}/instructorTypes", method = RequestMethod.POST, produces="application/json")
+    @ResponseBody
+    public InstructorType createLineItem(@PathVariable long budgetId,
+                                   @RequestBody InstructorType newInstructorType,
+                                   HttpServletResponse httpResponse) {
+        // Ensure valid params
+        Budget budget = budgetService.findById(budgetId);
+
+        if (budget == null) {
+            httpResponse.setStatus(HttpStatus.BAD_REQUEST.value());
```

Comment:
Should be 404 if the ID returns no record. Bad request implies the request wasn't understood or was malformed.